tests/test_legacy.py

```diff
@@ -34,128 +37,156 @@ def setUp(self):
                 ref_dataset_seq[0].Modality = ''
                 with self.assertRaises(ValueError):
-                    multiframe_item = LegacyConvertorClass(legacy_datasets=ref_dataset_seq,
-                                                           series_instance_uid=self._output_series_instance_uid,
-                                                           series_number=self._output_instance_number,
-                                                           sop_instance_uid=self._output_sop_instance_uid,
-                                                           instance_number=self._output_instance_number)
+                    LegacyConvertorClass(
+                        legacy_datasets=ref_dataset_seq,
+                        series_instance_uid=self._output_series_instance_uid,
+                        series_number=self._output_instance_number,
+                        sop_instance_uid=self._output_sop_instance_uid,
+                        instance_number=self._output_instance_number)
                 ref_dataset_seq[0].Modality = tmp_orig_modality
 
     def test_wrong_sop_class_uid(self):
         for m in self._modalities:
             with self.subTest(m=m):
-                LegacyConvertorClass = getattr(sop, "LegacyConvertedEnhanced" + m + "Image")
+                LegacyConvertorClass = \
+                    getattr(sop, "LegacyConvertedEnhanced" + m + "Image")
                 ref_dataset_seq = getattr(self, "_ref_dataset_seq_" + m)
                 tmp_orig_sop_class_id = ref_dataset_seq[0].SOPClassUID
                 ref_dataset_seq[0].SOPClassUID = '1.2.3.4.5.6.7.8.9'
                 with self.assertRaises(ValueError):
-                    multiframe_item = LegacyConvertorClass(legacy_datasets=ref_dataset_seq,
-                                                           series_instance_uid=self._output_series_instance_uid,
-                                                           series_number=self._output_instance_number,
-                                                           sop_instance_uid=self._output_sop_instance_uid,
-                                                           instance_number=self._output_instance_number)
+                    LegacyConvertorClass(
+                        legacy_datasets=ref_dataset_seq,
+                        series_instance_uid=self._output_series_instance_uid,
+                        series_number=self._output_instance_number,
+                        sop_instance_uid=self._output_sop_instance_uid,
+                        instance_number=self._output_instance_number)
                 ref_dataset_seq[0].SOPClassUID = tmp_orig_sop_class_id
 
     def test_mixed_studies(self):
         for m in self._modalities:
             with self.subTest(m=m):
-                LegacyConvertorClass = getattr(sop, "LegacyConvertedEnhanced" + m + "Image")
+                LegacyConvertorClass = \
+                    getattr(sop, "LegacyConvertedEnhanced" + m + "Image")
```

Comment:
```suggestion
                LegacyConvertorClass = getattr(
                    sop,
                    "LegacyConvertedEnhanced{}Image".format(m)
                 )
```

Reply:
When doing multi-line statements put closing bracket on new line and use a separate line for each argument.

Reply:
Use format strings instead of concatenating substrings using `+`

Reply:
Sure. Will do.